packages/appstore/src/components/modals/wallets-migration-failed/wallets-migration-failed.tsx

```diff
@@ -0,0 +1,115 @@
+
+const WalletsMigrationFailed = observer(() => {
+    const { traders_hub } = useStore();
+    const { is_wallet_migration_failed, setWalletsMigrationFailedPopup } = traders_hub;
+
+    const wallets_migration_failed_ref = React.useRef<HTMLDivElement>(null);
+
+    const handleClose = () => {
+        setWalletsMigrationFailedPopup(false);
+    };
+
+    const handLiveChatButtonClick = () => {
+        window.LC_API?.open_chat_window();
+        setWalletsMigrationFailedPopup(false);
+    };
+
+    const validateClickOutside = (e: MouseEvent) => {
+        return is_wallet_migration_failed && !wallets_migration_failed_ref?.current?.contains(e.target as Node);
+    };
+
+    useOnClickOutside(wallets_migration_failed_ref, handleClose, validateClickOutside);
+
+    return (
+        <React.Fragment>
+            {is_wallet_migration_failed && (
+                <React.Fragment>
+                    <DesktopWrapper>
+                        <Modal
+                            is_open={is_wallet_migration_failed}
+                            toggleModal={handleClose}
+                            width='44rem'
+                            should_header_stick_body={false}
+                            has_close_icon={false}
+                        >
+                            <div ref={wallets_migration_failed_ref}>
+                                <Modal.Body>
+                                    <Text
+                                        as='h1'
+                                        size='s'
+                                        color='prominent'
+                                        weight='bold'
+                                        className='wallets-migration-failed__title'
+                                    >
+                                        {localize('Sorry for the interruption')}
+                                    </Text>
+                                    <Text size='xs'>
+                                        {localize(
+                                            "We're unable to complete with the Wallet upgrade. Please try again later or contact us via live chat."
+                                        )}
+                                    </Text>
+                                </Modal.Body>
+                                <Modal.Footer className='wallets-migration-failed__footer'>
+                                    <Button secondary large onClick={handLiveChatButtonClick}>
+                                        {localize('Go to live chat')}
+                                    </Button>
+                                    <Button primary large onClick={handleClose}>
+                                        {localize('Back to Trader’s Hub')}
+                                    </Button>
+                                </Modal.Footer>
+                            </div>
+                        </Modal>
+                    </DesktopWrapper>
+                    <MobileWrapper>
```

Comment:
@sergei-deriv Can we avoid duplicating the whole component just to use `DesktopWrapper` and `MobileWrapper` please 🙏🏻 

Just use `is_mobile` from UI store instead 🙇🏻

Reply:
@farzin-deriv Done. Please, check again when you have time 🙏

Reply:
@sergei-deriv Awesome, Thank you 🌹🙇🏻